lib/puppet/provider/cron/filetype.rb

```diff
@@ -0,0 +1,209 @@
+require 'puppet/util/filetype'
+
+class Puppet::Provider::Cron
+  # This class defines the crontab filetypes
+  class FileType < Puppet::Util::FileType
```

Comment:
Not sure if this is the best solution since this is not a provider, but unfortunately it seems like this is the best place to put these filetypes based on what I'm reading in https://puppet.com/docs/puppet/5.3/modules_fundamentals.html#module-structure.

Reply:
I did the same for the windows msi and exe packages, eg https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/package/windows/exe_package.rb